docker/README.md

```diff
@@ -0,0 +1,54 @@
+* Docker installation instructions are at https://docs.docker.com/install/.
+* Docker Compose installation instructions are at https://docs.docker.com/compose/install/.
+* To run Docker commands as a regular user instead of as root (with sudo), follow the instructions at https://docs.docker.com/engine/install/linux-postinstall/.
+
+## Getting Started
+* Open a terminal with a shell.
+* Clone the repository. If you're planning on contributing code to the project (which we [LOVE](CONTRIBUTING.md)), begin by forking this repo with the Fork button in the top-right corner of this screen.
+* Use git clone to copy your fork onto your local machine.
+```sh
+$ git clone https://github.com/YOUR_GITHUB_USERNAME_HERE/openfoodnetwork
+```
+* Otherwise, if you just want to get things running, clone from the OFN main repo:
+
+```sh
+$ git clone git@github.com:openfoodfoundation/openfoodnetwork.git
+```
+* Go at the root of the app:
+
+```sh
+$ cd openfoodnetwork
+```
+* Download the Docker images, build the Docker containers, seed the database with sample data, AND log the screen output from these tasks:
+```sh
+$ docker/build
+```
+* Run the Rails server and its required Docker containers:
+
+```sh
+$ docker/server
+```
+* The default admin user is 'ofn@example.com' with the password 'ofn123'.
+* View the app in the browser at `http://localhost:3000`.
+* You will then get the trace of the containers in the terminal. You can stop the containers using Ctrl-C in the terminal.
+* You can find some useful tips and commands [here](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Docker:-useful-tips-and-commands).
+
+## Script Summary
+* docker/build: This script builds the Docker containers specified for this app, seeds the database, and logs the screen output for these operations.  After you use "git clone" to download this repository, run the docker/build script to start the setup process.
+* docker/cop: This script runs RuboCop.
+* docker/exec: Use this script to run commands within the Docker container.  If you want shell access, enter "docker/exec bash".  To execute "ls -l" within the Docker container, enter "docker/exec ls -l".
+* docker/nuke: Use this script to delete all Docker images and containers.  This fully resets your Docker setup and is useful for making sure that the setup procedure specified for this app is complete.
+* docker/seed: Use this script to seed the database.  Please note that this process is not compatible with simultaneously running the Rails server or tests.
+* docker/server: Use this script to run this app in the Rails server.  This script executes the "docker-compose up" command and logs the results.  If all goes well, you will be able to view this app on your local browser at http://localhost:3000/.
+* docker/test: Use this script to run the entire test suite.
+* docker/testc: Use this script to run just the controller tests.
+* docker/testm: Use this script to run just the model tests.
```

Comment:
nice!
can we order this in the order of typical usage?
- build
- server
- test
- exec
- seed
- nuke